src/frontend/Semantic_check.ml

```diff
@@ -1444,7 +1444,7 @@ and semantic_check_fundef_decl ~loc id body =
         if Symbol_table.check_is_unassigned vm id.name then
           error @@ Semantic_error.fn_decl_without_def loc
         else
-          let _ = Symbol_table.set_is_unassigned vm id.name in
+          let () = Symbol_table.set_is_unassigned vm id.name in
```

Comment:
This could also be `else ( Symbol_table.set_is_unassigned vm id.name ; ok () )` but this is fine too